doc/rados/operations/crush-map-edits.rst

```diff
@@ -629,29 +641,38 @@ There are three types of transformations possible:
    single bucket.  For example, in the previous example, we want the
    ``ssd`` bucket to be mapped to the ``default`` bucket.
 
-The final command to convert the map comprised of the above fragments would be something like::
+The final command to convert the map comprised of the above fragments would be something like:
+
+.. prompt:: bash $
 
-  $ ceph osd getcrushmap -o original
-  $ crushtool -i original --reclassify \
-      --set-subtree-class default hdd \
-      --reclassify-root default hdd \
-      --reclassify-bucket %-ssd ssd default \
-      --reclassify-bucket ssd ssd default \
-      -o adjusted
+  ceph osd getcrushmap -o original
+  crushtool -i original --reclassify \
+    --set-subtree-class default hdd \
+    --reclassify-root default hdd \
+    --reclassify-bucket %-ssd ssd default \
+    --reclassify-bucket ssd ssd default \
+    -o adjusted
 
-In order to ensure that the conversion is correct, there is a ``--compare`` command that will test a large sample of inputs to the CRUSH map and ensure that the same result comes back out.  These inputs are controlled by the same options that apply to the ``--test`` command.  For the above example,::
+In order to ensure that the conversion is correct, there is a ``--compare`` command that will test a large sample of inputs to the CRUSH map and ensure that the same result comes back out.  These inputs are controlled by the same options that apply to the ``--test`` command.  For the above example,:
+
+.. prompt:: bash $
+
+   crushtool -i original --compare adjusted
+
+::
 
-  $ crushtool -i original --compare adjusted
   rule 0 had 0/10240 mismatched mappings (0)
   rule 1 had 0/10240 mismatched mappings (0)
   maps appear equivalent
 
 If there were difference, you'd see what ratio of inputs are remapped
```

Comment:
@zdover23 This sentence is garbled.

Reply:
@colemitchell, no kidding. Much of this file could stand a refinement pass. I will create a pull request that addresses your comments in this PR (and I appreciate your grammar and usage notes, but this is a prompt-updating PR, so I won't be scope creeping it).

Reply:
@colemitchell: I have made all the corrections that you have suggested in this PR: https://github.com/ceph/ceph/pull/48362

Would you check the prompts in this PR, make sure that I have indeed rendered them unselectable, and then give me an approval? (Then, if Anthony hasn't beaten you to it, check out https://github.com/ceph/ceph/pull/48362.)

Reply:
@colemitchell: This PR should probably be merged before https://github.com/ceph/ceph/pull/48362, because I cherry-picked the commit in this PR into the branch I created for that PR. I won't bore you with the details, but it would really reduce the amount of unnecessary work that I'll have to do if you approve this one prior to approving that one.